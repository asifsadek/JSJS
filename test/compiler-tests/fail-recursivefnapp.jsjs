val z = /\(p) => p(p);
