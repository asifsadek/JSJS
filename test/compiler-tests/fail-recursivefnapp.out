Type error: expected value of type 'A', got a value of type '(A) -> B' instead
